e2e/kube.go

```diff
@@ -93,7 +93,7 @@ func ensureDebugDaemonsets(ctx context.Context, t *testing.T, kube *Kubeclient,
 func getDebugDaemonsetTemplate(t *testing.T, deploymentName, targetNodeLabel string, isHostNetwork, isAirgap bool) string {
 	image := "mcr.microsoft.com/cbl-mariner/base/core:2.0"
 	if isAirgap {
-		image = fmt.Sprintf("%s.azurecr.io/aks/cbl-mariner/base/core:2.0", config.PrivateACRName)
```

Comment:
Why is this able to be reverted? 

If so, can you remove the isAirgap variable from the function since it isn't being used.

Reply:
it is to rever the prefix `aks` in this pr

Reply:
I'm not sure I quite understand why removing the aks prefix means we don't need to be able to pull cbl-mariner using our ACR? This might be why you are seeing node not ready?

Reply:
Oh my bad, when I left this comment I was using a vscode extension that showed a different diff of the entire isAirgap statement removed.